xbmc/video/VideoInfoScanner.cpp

```diff
@@ -437,6 +452,44 @@ namespace VIDEO
     return !m_bStop;
   }
 
+  std::map<std::string, std::string> CVideoInfoScanner::UpdateSetFromLocalArtwork(
+      const CVideoInfoTag& setDetails)
+  {
+    CGUIListItem::ArtMap movieSetArt;
+    m_database.Open();
+    if (IsArtNeededAndLocalArtwork(setDetails))
+    {
+      // directory exists in Movie Set Information Folder and artwork currently blank
+      const std::string movieSetInfoPath = GetMovieSetInfoFolder(setDetails.m_strTitle);
+      const std::vector<std::string> movieSetArtTypes =
+          CVideoThumbLoader::GetArtTypes(MediaTypeVideoCollection);
+      AddLocalItemArtwork(movieSetArt, movieSetArtTypes, movieSetInfoPath, true, false);
+      m_database.SetArtForItem(setDetails.m_iDbId, MediaTypeVideoCollection, movieSetArt);
+    }
+    m_database.Close();
+    return movieSetArt;
```

Comment:
It doesn't look like the return value is actually used by the callers and a true/false boolean would be enough?